singleheader/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(SINGLEHEADER_FILES
   ${CMAKE_CURRENT_BINARY_DIR}/simdjson.h
   ${CMAKE_CURRENT_BINARY_DIR}/amalgamate_demo.cpp
   ${CMAKE_CURRENT_BINARY_DIR}/README.md
+  ${CMAKE_CURRENT_SOURCE_DIR}/done.txt
```

Comment:
Should be CMAKE_CURRENT_BINARY_DIR, I think. I'll integrate this into my PR ...

Reply:
I'm going to see what happens if I don't include done.txt. Theoretically at least, the other files should always get updated.

Reply:
Yes. But you have to respect the rule that only one target should directly depend on the custom command. If you do so, it should work.